api/models.py

```diff
@@ -135,6 +135,8 @@ class Meta:
     # If this is an image, it is sent to ocr.space for automatic transcription.
     content_url = models.URLField(null=True, blank=True)
 
+    nsfw = models.BooleanField(null=True, blank=True)
```

Comment:
Maybe a small comment for this would be nice, for example explaining how this is determined (that it's based on the original post on Reddit, and not updated after we pulled the post)